background.js

```diff
@@ -1,14 +1,40 @@
 chrome.runtime.onInstalled.addListener(function initialization(){
 	turnFilteringOff();
-	var blockedSites = ["://www.onet.","://www.wp."];
-	chrome.storage.sync.set({'blockedSites': blockedSites}, function() {
-			console.log('Blocked sites are loaded.');
-	});
-	chrome.storage.sync.set({'blockingMethod': "close_tab"}, function() {});
+
+	chrome.storage.sync.set({'blockingMethod': "close_tab"});
 	let timerData = { isTimerEnabled: false, blockUntilMilliseconds: 0};
-	chrome.storage.sync.set({'timerData': timerData}, function() {});
+	chrome.storage.sync.set({'timerData': timerData});
+	
+	chrome.storage.sync.get('blockedSites', function(data) {
+		blockedSites = data.blockedSites;
+		if(typeof blockedSites != "undefined" && blockedSites != null 
+			&& blockedSites.length != null && blockedSites.length > 0){
+			var defaultListConfirm = confirm("We detected that our extension" 
+				+ " was installed once on this device.\nDo you want to load your old filter list?");
```

Comment:
We have (albo We've) detected that out extension was once installed on this device